demo/index_facets.R

```diff
@@ -0,0 +1,25 @@
+# Xapian provides functionality which fascilitates dynamic generation of complete lists of values which feature in matching documents.
+# This example shows how to ensure that the values you wish to use in facets are stored as document values.
+
+library(RXapian)
+
+# preparing arguments for xapian_index()
+dataFile <- system.file("sampleData/100-objects-v1.csv", package="RXapian")
+
+data <- read.csv(dataFile)
+
+db<- tempfile(pattern="RXapianEx-")
+
+id<-c(0)
+
+indexFields<-list(list(index=2,prefix="S"),
```

Comment:
Can this be done using names instead of indexes, to make the demo clearer? Or alternatively a comment, so people don't have to have the CSV open at the same time to understand what's going on. (Similarly a comment would be good for `id<-c(0)` above, which is a little terse for people to understand first time round.)